.github/workflows/test.yml

```diff
@@ -220,3 +220,6 @@ jobs:
 
     - name: Test with tox
       run: tox -e docs,style,readme
```

Comment:
I think it's more appropriate for this check of inline styles or scripts to exist in the `Makefile`, then including it here with the other static checks run by tox.

Doing so will also make it easier for developers to run this check before pushing to github and having CI catch it.